docs/source/notebooks/bayesian_estimation_of_ode_parameters.ipynb

```diff
@@ -0,0 +1,596 @@
+{
```

Comment:
<p>Parameters is misspelled paramerters'</p><p> </p><p>Resistance is mispelled restistance</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3578/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>